src/v3/src/transformer/password/__snapshots__/transformResetPasswordAuthenticator.test.ts.snap

```diff
@@ -112,7 +125,17 @@ Object {
     },
     "fieldsToExclude": [Function],
     "fieldsToTrim": Array [],
-    "fieldsToValidate": Array [],
+    "fieldsToValidate": Array [
+      "confirmPassword",
+      "passwordMatchesValidation",
+      "confirmPassword",
+      "passwordMatchesValidation",
+      "confirmPassword",
+      "passwordMatchesValidation",
```

Comment:
Hmm, I wonder why this is happening

Reply:
Ahh, its because in the beforeEach, we are not resetting the `dataSchema.fieldsToValidate` array (we usually dont have to do that because it is not used much, if at all except here, in this layer).

Reply:
Ahh okay, so it's only happening in the test. I will update the `beforeEach`.